src/olympia/migrations/907-drop-password-and-confirmationcode.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE users
+    DROP COLUMN `password`,
+    DROP COLUMN `confirmationcode`;
```

Comment:
Please add a note to the push etherpad when you merge this, so that we can anticipate any issues